docs/user/azure/customization.md

```diff
@@ -102,7 +102,8 @@ platform:
   azure:
     region: centralus
     baseDomainResourceGroupName: os4-common
-    osDisk:
+    defaultMachinePlatform:
+      osDisk:
```

Comment:
My preference is to remove this entirely. It doesn't really make sense to stipulate defaults for the OS disk. Moreover, these fields have no effect on this install config anyway since the values are explicitly specified for the master and workers.